website_sale_b2c_v10/README.rst

```diff
@@ -0,0 +1,82 @@
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
+   :alt: License: LGPL-3
+
+===================
+e-Commerce B2C mode
+===================
+
+This module extends the functionality of ``website_sale`` to support B2C
+pricing (show prices with taxes included).
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+#. Go to *Sales > Configuration > Settings > Quotations & Sales > Sale Price*.
+#. Enable *Show line subtotals with taxes included (B2C)*.
+#. *Apply*.
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Follow steps in the *Configuration* section above.
+#. Edit any product.
+#. Go to *Accounting* tab.
+#. Set some *Customer Taxes* to it.
+#. Publish the product in the website.
+#. You will notice the price that appears in the website is computed with
+   taxes included.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/113/9.0
+
+Known issues / Roadmap
+======================
+
+* No tests.
+* When upgrading your database to Odoo 10.0, you will have to uninstall this
```

Comment:
This is not needed if migration scripts are provided on OpenUpgrade, so just put "This module is not needed on v10 and without proper migration scripts, a version 9 upgraded DB can have problems".